wger/core/urls.py

```diff
@@ -39,7 +39,7 @@
 
 # sub patterns for user
 patterns_user = [
-    url(r'^login$', user.login, name='login'),
+    url(r'^login', user.login, name='login'),
```

Comment:
why is the $ sign removed

Reply:
$ Added. This was removed to manage some errors during redirection.